java/src/main/native/src/row_conversion.cu

```diff
@@ -2258,10 +2258,17 @@ std::unique_ptr<table> convert_from_rows(lists_column_view const &input,
       if (schema[i].id() == type_id::STRING) {
         // stuff real string column
         auto string_data = string_row_offset_columns[string_idx].release()->release();
-        output_columns[i] = make_strings_column(num_rows, std::move(string_col_offsets[string_idx]),
-                                                std::move(string_data_cols[string_idx]),
-                                                std::move(*string_data.null_mask.release()),
-                                                cudf::UNKNOWN_NULL_COUNT);
+        auto const null_count = [&] {
+          auto const &null_mask = *string_data.null_mask;
+          return null_mask.data() ?
+                     cudf::null_count(static_cast<bitmask_type const *>(null_mask.data()), 0,
+                                      num_rows) :
+                     0;
+        }();
```

Comment:
Probably this is just this?
```
auto const null_count = string_row_offset_columns[string_idx]->null_count();
```

Reply:
Agreed. Fixed.

Note: As per discussion with @hyperbolic2346, this code is likely cruft. I'll raise a separate issue to have `row_conversion.*` removed from CUDF.